### ISSUE TITLE
global variables e.g. user.login stop expanding when change time range

### ISSUE THREAD

**Issue by allomorphy** [grafana/grafana#25359]
**What happened**:
The global variable stopped expanding when adjusting the time range.

**What you expected to happen**:
The variable continued to expand to its value in normal usage.

**How to reproduce it (as minimally and precisely as possible)**:
This occured in both Safari and Chrome on Mac OS and Chrome on iOS
It happened when the time range was changed either with the drop down or with the drag zoom control

On Mac OS, it almost always corrects when you log out and then log back in again
On iOS Chrome, it fixed also when selecting another dashboard and then reselecting the dashboard that includes the global var.

**Anything else we need to know?**:

**Environment**:
- Grafana version: Grafana v7.1.0-db3b38efpre (467348bded)
- Data source type & version: InfluxDB v1.7.0
- OS Grafana is installed on: linux in container :master tag
- User OS & Browser: Mac Safari/Chrome iOS Chrome
- Grafana plugins: GF_INSTALL_PLUGINS=marcuscalidus-svg-panel,blackmirror1-singlestat-math-panel,briangann-gauge-panel,jdbranham-diagram-panel,natel-plotly-panel"
- Others:

**Comment by hugohaggmark** [comment]
@allomorphy would you say that this https://github.com/grafana/grafana/issues/23841 is kind of the same issue (except that issue is on refresh dashboard)

**Comment by allomorphy** [comment]
Yes @hugohaggmark❗ 
The top right dashboard refresh recycle icon, causes the variable to no longer expand and the "no data" error. So far identical behaviour to that of changing the `time range`.

I noticed something that I did not determine last time, that the actual browser refresh with the modifier-R, this causes the variable to expand correctly again. So this is a reproducible correcting mechanism.

FWIW, this issue also affects variables that have been defined for some time, like for example $__dashboard.

HTH
Best wsihes
Eric
